### DISCUSSION TITLE
Ephemeral models, one more time.

### DISCUSSION THREAD

**Discussion by Kamal-Klm** [dagster-io/dagster#16165] [Q&A] [answered]
Dear team,
I can see that you well responded to feature requests to disable the rendering of ephemeral dbt models. I suppose it fits dagster's view on what qualifies as a data asset. 
I'd appreciate if I'm directed to away (if it exists) to render ephemerals in the graph ? (I can see the source code of dagster_dbt.asset_utils filtering those out).
**Any prospects of an option for the user to decide on including those?**

Thank you.

**Comment by sryza** [comment]
@rexledesma - thoughts?

**Comment by rexledesma** [comment, marked as answer]
We're not looking to render ephemeral models in the Dagster asset graph at this time.
